.github/agents/state-docs-keeper.agent.md

```
@@ -0,0 +1,33 @@
+
+Output format
+## 📚 State + Docs Pass (adze)
+
+### Files updated
+- <path>: <what changed>
+
+### Claims verified (commands)
+- <claim> → `<command>`
+
+### Open TODOs (bounded)
+- [ ] ...
```

markdownlint-cli2 (0.21.0)

[warning] Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)